debugfs.ocfs2/dump_fs_locks.c

```diff
@@ -271,14 +278,26 @@ static int dump_version_two_and_three(FILE *file, FILE *out, int v3)
 		avg_exmode = total_exmode/num_exmode;
 
 	fprintf(out, "PR > Gets: %llu  Fails: %u    Waits Total: %lluus  "
-		"Max: %uus  Avg: %lluns\n",
+		"Max: %uus  Avg: %lluns",
 		num_prmode, num_prmode_failed, total_prmode/NSEC_PER_USEC,
 		max_prmode, avg_prmode);
+	if (version > 3) {
+		fprintf(out, " Last: %lluus", last_prmode);
+	}
+	fprintf(out, "\n");
 	fprintf(out, "EX > Gets: %llu  Fails: %u    Waits Total: %lluus  "
-		"Max: %uus  Avg: %lluns\n",
+		"Max: %uus  Avg: %lluns",
 		num_exmode, num_exmode_failed, total_exmode/NSEC_PER_USEC,
 		max_exmode, avg_exmode);
-	fprintf(out, "Disk Refreshes: %u\n", num_refresh);
+	if (version > 3) {
+		fprintf(out, " Last: %lluus", last_exmode);
+	}
+	fprintf(out, "\n");
+	fprintf(out, "Disk Refreshes: %u", num_refresh);
+	if (version > 3) {
+		fprintf(out, " Wait: %lluus", wait);
```

Comment:
Better to clarify it as "First Wait".
Other looks good.

Reply:
Done. Squashed.